### PR TITLE
5-It's not support read-only mode

### LINKED ISSUE #5: It's not support read-only mode.
```
Error: To enable read-only mode all connected tools should support it. Tools breakLine don't support read-only mode.
```

### PR DESCRIPTION
**Pull request by MrJuanGaviriaK**
Closes #5

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -145,4 +145,13 @@ export default class BreakLine {
       title: 'Break Line',
     };
   }
+
+  /**
+   * Returns true to notify the core that read-only mode is supported
+   *
+   * @return {boolean}
+   */
+  static get isReadOnlySupported() {
```
Comment (juanhurtado10) [inline src/index.js:154]: @MrJuanGaviriaK this is ok 👌   the tool will notify the `editor.js` **core** that the `readOnly` mode is supported. 

I think there is a missing piece, the `render()` method should not perform an action when this mode is active. 

What do you think?

Comment (juanhurtado10) [inline src/index.js:154]: @MrJuanGaviriaK Could you please add the corresponding tests cases? 

Thank you!

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (MrJuanGaviriaK) [reply src/index.js:154]: @juanhurtado10 

I agree, I'm going to work on it!!!

Thanks

**Comment by MrJuanGaviriaK** [comment]
Hi @juanhurtado10

I added some new tests to read-only mode:

Screen Shot 2021-08-23 at 8 29 17 PM

Best,

**Review by juanhurtado10 — APPROVED** [review]

**Comment by juanhurtado10** [comment]
@MrJuanGaviriaK Awesome thanks for contributing 🙌

**Pull request merged**